pootle/apps/accounts/utils.py

```diff
@@ -30,3 +33,106 @@ def get_user_by_email(email):
             return User.objects.get(email__iexact=email)
         except User.DoesNotExist:
             return None
+
+
+def merge_user(src_user, target_user):
+    """Merges one user to another
```

Comment:
Yes they do. Refer to our styleguide http://docs.translatehouse.org/projects/translate-toolkit/en/latest/developers/styleguide.html#docstrings and PEP 257 https://www.python.org/dev/peps/pep-0257/#one-line-docstrings